client/app/components/Parameters.jsx

```diff
@@ -43,13 +43,23 @@ export default class Parameters extends React.Component {
     appendSortableToParent: true,
   };
 
+  toCamelCase = str => {
+    if (isEmpty(str)) {
+      return "";
+    }
+    return str.replace(/\s+/g, "").toLowerCase();
```

Comment:
It seems unrelated to the implementation and function name?

Reply:
Cookl

Reply:
Seems like this `toCamelCase()` function should be removed from the PR.